addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1624,6 +1624,21 @@ options.registry.anchor = options.Class.extend({
     },
 });
 
+/**
+ * Controls box properties.
+ */
+options.registry.Box = options.Class.extend({
+    /**
+     * @override
+     */
+    selectClass(previewMode, widgetValue, params) {
+        this._super(...arguments);
+        if (!widgetValue && params.activeValue.includes('shadow')) {
```

Comment:
@ThanhDodeurOdoo You did not change that correctly

Reply:
It seems to work as intended. From your comment above you suggested `params.possibleValues.includes('shadow')` but possibleValues is an array so it wouldn't be
true for the classes you mentioned (shadow-sm/shadow-lg).

`params.possibleValues.some(s => s.includes('shadow'))`?

Reply:
Oh ok, I was not sure `includes` existed on string. Anyways, yes:
`params.possibleValues.some(s => s === 'shadow')`
No need of includes here, you only need to check the exact match exists as possible value.

Reply:
Oh no I see, then array.includes is ok for exact match then, I thought the use case was to have `params.possibleValues` like `['', 'shadow']` or `['', 'shadow-sm']` for which is should return true. But now I see that all `possibleValues` that would include either `shadow-sm` or `shadow-lg` should also include `shadow`.